src/main/java/frc/trigon/robot/constants/CommandConstants.java

```diff
@@ -14,6 +14,7 @@
 import java.awt.*;
 
 public class CommandConstants {
+    public static boolean IS_CLIMBING = false;
```

Comment:
Why do you need this? (There is a reason that you did not implement. Make sure to know exactly why you have each piece of code you're writing. Or maybe make sure to code the command first by yourself, and then see what we had last year.)